integration-tests/emulated/chains/parachains/bridges/bridge-hub-polkadot/src/genesis.rs

```diff
@@ -22,6 +22,7 @@ use emulated_integration_tests_common::{
 };
 use parachains_common::Balance;
 
+pub const ASSETHUB_PARA_ID: u32 = 1000;
```

Comment:
Just a preference, I would separate them:
```suggestion
pub const ASSET_HUB_PARA_ID: u32 = 1000;
```

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/130/commits/0c6e184afa829c55e242623fb923defff61dcf2c.